### PR TITLE
Sport programming Odintsovo - closes #158

### LINKED ISSUE #158: перенести новость в блог (спортивное программирование) [0.5h]
https://odin.mgimo.ru/nov-pod-mgimo/3477-student-mgimo-odintsovo-prinyal-uchastie-v-sborakh-po-sportivnomu-programmirovaniyu

Комментарии:
- определить год в дате, использовать во front matter в блоге
- футер новости - Оригинал на сайте МГИМО-Одинцово <ссылка>
- текст новости показать здесь в ишью в верстке
- прогнать через `npx prettier`
- PR в ветку preview, референсить ишью в коммитах и в PR

### PR DESCRIPTION
**Pull request by edyatl**
News transfer https://odin.mgimo.ru/nov-pod-mgimo/3477-student-mgimo-odintsovo-prinyal-uchastie-v-sborakh-po-sportivnomu-programmirovaniyu
Closes the issue #158

### CODE REVIEW

File: content/ru/blog/2020-09-student-mgimo-odintsovo-prinyal-uchastie-v-sborakh-po-sportivnomu-programmirovaniyu/index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+title: "Студент МГИМО-Одинцово принял участие в сборах по спортивному программированию"
+date: 2020-09-16
+lastmod: 2022-07-04
+draft: false
+slug: student-mgimo-odintsovo-prinyal-uchastie-v-sborakh-po-sportivnomu-programmirovaniyu
+---
+
+{{< img src="student-mgimo-odintsovo2.jpg"  caption="Студент МГИМО-Одинцово принял участие в сборах по спортивному программированию" width=100% >}}
+
+С 5 по 12 сентября в рамках образовательного проекта Moscow Workshops, реализуемого на базе кампуса МФТИ, прошли онлайн-сборы по спортивному программированию [«Discover the World 2020»](https://discover.it-edu.com/).
```
Comment (epogrebnyak) [inline content/ru/blog/2020-09-student-mgimo-odintsovo-prinyal-uchastie-v-sborakh-po-sportivnomu-programmirovaniyu/index.md]: Предлагаю указать год в тексте. 2020?

File: content/ru/blog/2020-09-student-mgimo-odintsovo-prinyal-uchastie-v-sborakh-po-sportivnomu-programmirovaniyu/index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+В начале сентября студент программы Информационные технологии направления «Бизнес-информатика» Иван Мельников, капитан команды **BIT-ART**, принял участие в тренировочных сборах по спортивному программированию в рамках образовательного проекта **[Moscow Workshops](https://discover.it-edu.com/) (MW)** . В течение пяти дней участники слушали лекции по различным аспектам математики и программирования, тренировались решать олимпиадные задачи и разбирали допущенные ошибки.
+
+{{< img src="student-mgimo-odintsovo1.jpg"  caption="Иван Мельников" width=100% >}}
+
+Иван так прокомментировал результаты мероприятия: «Такие сборы серьёзно развивают логику и нестандартное мышление, которые являются наиболее важными качествами и в соревнованиях и вообще в условиях современных реалий. За всеми новшествами, да еще и на нескольких языках программирования, успеть невозможно, зато фундаментальные знания будут актуальны всегда и помогут значительно быстрее разобраться в интересующей технологии.»
+
+{{< img src="student-mgimo-odintsovo3.jpg"  caption="призываем всех заинтересованных студентов принять участие" width=100% >}}
```
Comment (epogrebnyak) [inline content/ru/blog/2020-09-student-mgimo-odintsovo-prinyal-uchastie-v-sborakh-po-sportivnomu-programmirovaniyu/index.md]: Приглашаем скорее,чем призываем

File: content/ru/blog/2020-09-student-mgimo-odintsovo-prinyal-uchastie-v-sborakh-po-sportivnomu-programmirovaniyu/index.md (hunk 1 above)
Comment (edyatl) [reply content/ru/blog/2020-09-student-mgimo-odintsovo-prinyal-uchastie-v-sborakh-po-sportivnomu-programmirovaniyu/index.md]: fixed in new commit https://github.com/finec-mgimo/finec-mgimo.github.io/pull/174/commits/9c943f31df4758a213dde46f555017cb597ab2c2

File: content/ru/blog/2020-09-student-mgimo-odintsovo-prinyal-uchastie-v-sborakh-po-sportivnomu-programmirovaniyu/index.md (hunk 2 above)
Comment (edyatl) [reply content/ru/blog/2020-09-student-mgimo-odintsovo-prinyal-uchastie-v-sborakh-po-sportivnomu-programmirovaniyu/index.md]: fixed in new commit https://github.com/finec-mgimo/finec-mgimo.github.io/pull/174/commits/9c943f31df4758a213dde46f555017cb597ab2c2

**Pull request merged**